File: iclean-api/src/main/java/com/iclean/icleanapi/Contratante.java
package com.iclean.icleanapi;

import java.util.Date;
import java.util.List;

public class Contratante extends Usuario {
    private List<Imovel> listaImoveis;

    public Contratante(String nome, String cpf, String telefone, Date dataNascimento, List<Avaliacao> listaAvaliacoes, List<Imovel> listaImoveis) {
        super(nome, cpf, telefone, dataNascimento, listaAvaliacoes);
        this.listaImoveis = listaImoveis;
    }

    public List<Imovel> getListaImoveis() {
        return listaImoveis;
    }
}

File: IcleanMobile/app/src/main/java/com/example/icleanmobile/Contratante.kt
package com.example.icleanmobile

data class Contratante (
    var id: Int?,
    val nome: String,
    val cpf: String,
    val telefone: String,
    val dataNascimento: String,
    val email: String,
    val senha: String,
    val autentiacado: Boolean?
)